force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -83,7 +109,15 @@ public with sharing class FlowControlLoopsCollections {
      * @return A string indicating whether the number is "Even" or "Odd", or null if the number is null.
      */
     public static String checkEvenOdd(Integer a) {
-        return null; // Replace null with the variable you used to store the result
+        String result = null;
+        if (a == null) {
+            return result;
+        } else if (Math.mod(a, 2) == 0) {
+            result= 'Even';
+          } else {
+            result = 'Odd';
+          }
+        return result; 
```

Comment:
Good work. If you want to explore try solving this a ternary operator. That is how I like to approach these.